src/permissionGroups/components/PermissionGroupDeleteDialog/PermissionGroupDeleteDialog.stories.tsx

```diff
@@ -16,4 +16,10 @@ const props: PermissionDeleteDialogProps = {
 
 storiesOf("Views / Permission Groups / Permission Group Delete", module)
   .addDecorator(Decorator)
-  .add("remove single", () => <PermissionGroupDeleteDialog {...props} />);
+  .add("remove single", () => <PermissionGroupDeleteDialog {...props} />)
+  .add("Got error", () => (
+    <PermissionGroupDeleteDialog
+      {...props}
+      error={"Something went realy wrong."}
```

Comment:
Can't we display a more meaningful error informing what could be the potential cause? I think we should know it based on the error code somehow?

Reply:
Thats just example for storybook. In app msg is formatted by already created utils with better msgs.
Besides that - to avoid such error user is not presented with `delete` button when not having enough permissions.